data/ch/testimonial.yml

```diff
@@ -0,0 +1,26 @@
+################################# Testimonial #########################
+testimonial:
+  enable : true
+  bg_image : images/backgrounds/17-042_main_image.jpg
+  bg_image_webp : images/backgrounds/17-042_main_image.webp
+  testimonial_item :
+    # testimonial item loop
+    - name : Jonathon Andrew
+      image : images/client/client-1.jpg
+      image_webp : images/client/client-1.webp
+      date : Dec 26, 2018
+      content : Lorem ipsum dolor sit amet, consectetur adipisicing elit. Praesentium minima ex, quia error, deserunt voluptatibus repellendus? Incidunt eligendi aliquam repellat molestias, vel sed optio rem odit quis ratione non consequuntur!
+
+    # testimonial item loop
+    - name : Mark Dinn
+      image : images/client/client-2.jpg
+      image_webp : images/client/client-2.webp
+      date : Dec 26, 2018
+      content : Lorem ipsum dolor sit amet, consectetur adipisicing elit. Praesentium minima ex, quia error, deserunt voluptatibus repellendus? Incidunt eligendi aliquam repellat molestias, vel sed optio rem odit quis ratione non consequuntur!
+
+    # testimonial item loop
+    - name : Lemos Kail
+      image : images/client/client-1.jpg
+      image_webp : images/client/client-1.webp
+      date : Dec 26, 2018
+      content : Lorem ipsum dolor sit amet, consectetur adipisicing elit. Praesentium minima ex, quia error, deserunt voluptatibus repellendus? Incidunt eligendi aliquam repellat molestias, vel sed optio rem odit quis ratione non consequuntur!
```

Comment:
We can remove this file (we'll add Chinese-language in a future PR using the go-18n for Hugo approach)